tests/benchdnn/inputs/conv/option_gpu_ci

```diff
@@ -22,12 +22,14 @@
 ## Type combinations
 --dir=FWD_I
 --dt=s8:s8:f32,s8:s8:bf16,s8:s8:f16,s8:s8:s32,s8:s8:s8,s8:s8:u8, \
-      u8:s8:f32,u8:s8:bf16,u8:s8:f16,u8:s8:s32,u8:s8:s8,u8:s8:u8
+      u8:s8:f32,u8:s8:bf16,u8:s8:f16,u8:s8:s32,u8:s8:s8,u8:s8:u8, \
+      s8:u8:f32,s8:u8:bf16,s8:u8:f16,s8:u8:s32,s8:u8:s8,s8:u8:u8, \
+      u8:u8:f32,u8:u8:bf16,u8:u8:f16,u8:u8:s32,u8:u8:s8,u8:u8:u8
 --batch=shapes_ci_gpu
 
 ## Attributes
 --dir=FWD_I
---dt=u8:s8:s32
+--dt=u8:s8:s32,s8:u8:s32,u8:u8:s32
```

Comment:
I'd probably keep u8:u8 only as it seems s8:u8 is an extreme case in real models.